Sources/Nitrozen-SwiftUI/Elements/Stepper/NitrozenStepperView.swift

```diff
@@ -0,0 +1,163 @@
+        case .systemImage(let imageName):
+            incremenButton(imageType: .systemImage(name: imageName))
+        case .assetImage(let imageName):
+            incremenButton(imageType: .assetImage(name: imageName))
+        }
+    }
+    
+    @ViewBuilder
+    func incremenButton(imageType:NitrozenAppearance.StepperView.CustomView) -> some View{
+        let borderColor = (maxNumber == Int(inputValue).or(0)) ?
+        appearance.actionButton.borderColorDisabled : appearance.actionButton.borderColor
+        
+        Button(action: {
+            increment()
+        }) {
+            Group{
+                switch imageType {
+                case .systemImage(let name):
+                    Image(systemName: name)
+                case .assetImage(let name):
+                    Image(name)
+                }
+            }
+            .frame(
+                width: appearance.actionButtonSize.height,
+                height: appearance.actionButtonSize.height
+            )
+        }
+        .foregroundColor(appearance.actionButton.titleColor)
+        .disabled(maxNumber == Int(inputValue).or(0))
+        .background(appearance.actionButton.backgroundColor)
+        .apply(
+            shape: appearance.viewShpae,
+            color:borderColor,
+            lineWidth: appearance.actionButton.borderWidth
+        )
+        
+    }
+    
+    @ViewBuilder
+    func decrementAction() -> some View{
+        switch appearance.decrementButton {
+        case .systemImage(let imageName):
+            decrementButton(imageType: .systemImage(name: imageName))
+        case .assetImage(let imageName):
+            decrementButton(imageType: .assetImage(name: imageName))
+        }
+    }
+    
+    @ViewBuilder
+    func decrementButton(imageType:NitrozenAppearance.StepperView.CustomView) -> some View{
+        let borderColor = (0 == Int(inputValue).or(0)) ?
+        appearance.actionButton.borderColorDisabled : appearance.actionButton.borderColor
+        
+        Button(action: {
+            decrement()
+        }) {
+            Group{
+                switch imageType {
+                case .systemImage(let name):
+                    Image(systemName: name)
+                case .assetImage(let name):
+                    Image(name)
+                }
+            }
+            .frame(
+                width: appearance.actionButtonSize.height,
+                height: appearance.actionButtonSize.height
+            )
+        }
+        .foregroundColor(appearance.actionButton.titleColor)
+        .background(appearance.actionButton.backgroundColor)
+        .disabled(0 == Int(inputValue).or(0))
+        .apply(
+            shape: appearance.viewShpae,
+            color:borderColor,
+            lineWidth: appearance.actionButton.borderWidth
+        )
+    }
+    
+    func increment(){
+        if Int(inputValue).or(0) < maxNumber{
+            var value = Int(inputValue).or(0)
+            value += 1
```

Comment:
Also consider a case where use needs to increment step value by 5,
e.g. start with 10 and end with 100 i.e. 10...100, and step will be incremented by 5 -- i.e. 10-15-20-25-....100